modules/xmpp/xmpp.js

```diff
@@ -713,16 +720,16 @@ export default class XMPP extends Listenable {
 
     /**
      * A private message is received, message that is not addressed to the muc.
-     * We expect private message coming from speaker stats component if it is
+     * We expect private message coming from plugins component if it is
      * enabled and running.
      *
      * @param {string} msg - The message.
      */
     _onPrivateMessage(msg) {
         const from = msg.getAttribute('from');
 
-        if (!this.speakerStatsComponentAddress
-            || from !== this.speakerStatsComponentAddress) {
+        if (!((from === this.speakerStatsComponentAddress)
+            || (from === this.conferenceDurationComponentAddress))) {
```

Comment:
Doesn't lint complain about the extra brackets?

Reply:
Or just simplify it a bit ...

Reply:
Removed the extra brackets